src/Installers/Windows/RuntimeSiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(RepositoryRoot)\.deps\dependencies.g.props" />
+  <Import Project="..\..\Directory.Build.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <PackageType>AzureSiteExtension</PackageType>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IncludeSymbols>false</IncludeSymbols>
+    <IncludeSource>false</IncludeSource>
+    <ContentTargetFolders>content</ContentTargetFolders>
+    <!--
+        NuGet skips files ending .nuget producing warning at the same time
+        there is no way to disable default excludes from MsBuild (see:https://github.com/NuGet/Home/issues/6450)
+        and we don't care about item templates on Antares
+     -->
+    <NoWarn>$(NoWarn);NU5119</NoWarn>
+
+    <TrimmedVersion>$(VersionPrefix.Substring(0, $(VersionPrefix.LastIndexOf('.'))))</TrimmedVersion>
+    <Title>ASP.NET Core $(TrimmedVersion) ($(Platform)) Runtime </Title>
+    <Description>This site extension installs Microsoft.AspNetCore.All, Microsoft.AspNetCore.App and Microsoft.NetCore.App shared runtimes.</Description>
+    <PackageTags>aspnetcore;AzureSiteExtension</PackageTags>
+    <PackageId>AspNetCoreRuntime.$(TrimmedVersion).$(Platform)</PackageId>
+
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(SignType)' != '' ">
+    <SignOutput>True</SignOutput>
```

Comment:
This property only works on .wixproj. In order to code sign a nupkg, we'll need to add a new step to repo.targets. See https://github.com/aspnet/AspNetCore/blob/master/build/CodeSign.targets